app/models/designation.js

```diff
@@ -151,6 +165,12 @@ export default Model.extend({
       return "appointment";
     } else if (this.get("isOnlineOrder")) {
       return "online_order";
+    } else if (this.get("isShipmentOrder")) {
+      return "shipment";
+    } else if (this.get("isStockitLocalOrder")) {
+      return "stockit_local_order";
+    } else if (this.get("isCarryOutOrder")) {
+      return "carry_out";
```

Comment:
Should this computed property also watch for `bookingType` and `detailType` along with `orderTransport` ?

```js
stateText: Ember.computed("orderTransport", "bookingType", "orderTransport",  function() {
```

Reply:
Yes, Thanks  :+1: